RecCalorimeter/src/components/CorrectCaloClusters.h

```diff
@@ -177,6 +214,34 @@ class CorrectCaloClusters : public GaudiAlgorithm {
   /// Downstream correction formulas in ROOT notation
   Gaudi::Property<std::vector<std::vector<std::string>>> m_downstreamFormulas {
       this, "downstreamFormulas", {}, "Downstream formulas (in ROOT notation)"};
+/// Benchmark method correction parameters (a, b, c, ...)
+  Gaudi::Property<std::vector<std::vector<double>>> m_benchmarkParametrization {
+      this, "benchmarkParametrization", {}, "Benchmark formula parameters"};
+  /// Benchmark correction formulas in ROOT notation
+  Gaudi::Property<std::vector<std::vector<std::string>>> m_benchmarkFormulas {
+      this, "benchmarkFormulas", {}, "Benchmark formulas (in ROOT notation)"};
+
+  /// Possibility to use different parametrization for very low energies (<10GeV) for benchmark method 
+  /// If true, the second formula in the input vector will be used for energies below m_eneSwitch 
+  Gaudi::Property<bool> m_benchmarkTwoFormulas{this, "benchmarkTwoFormulas", false};
+
+  /// For the energy (GeV) below this given threshold, the second energy formula in benchmarkFormulas is used 
+  /// To be used together with setting m_benchmarkTwoFormulas to true 
+  Gaudi::Property<double> m_benchmarkEneSwitch {
+      this, "benchmarkEneSwitch", {5.}, "Energy threshold in GeV to use different parametrization"};
```

Comment:
Can you merge this into one property? I mean is the second formula set won't be used, just set eneSwith below zero.